lib/Conversion/HandshakeToFIRRTL/HandshakeToFIRRTL.cpp

```diff
@@ -1356,7 +1433,8 @@ struct HandshakeFuncOpLowering : public OpConversionPattern<handshake::FuncOp> {
       // be instantiated in the top-module.
       else if (op.getDialect()->getNamespace() != "firrtl") {
         FModuleOp subModuleOp = checkSubModuleOp(topModuleOp, &op);
-        bool hasClock = isa<handshake::BufferOp, handshake::ControlMergeOp>(op);
+        bool hasClock =
```

Comment:
This list is going to keep growing with memory support. Not a requirement for this PR, but at some point maybe we could have a trait on the ops themselves to indicate if they have a clock.

Reply:
Good point!